bindings/rust/bench/src/harness.rs

```diff
@@ -64,6 +64,25 @@ pub trait TlsBenchHarness: Sized {
 
     /// Get whether or negotiated version is TLS1.3
     fn negotiated_tls13(&self) -> bool;
+
+    /// Send data from one connection
+    fn send(&mut self, sender: Mode, data: &[u8]) -> Result<(), Box<dyn Error>>;
+
+    /// Receive data sent to a connection
+    fn recv(&mut self, receiver: Mode, data: &mut [u8]) -> Result<(), Box<dyn Error>>;
+
+    /// Transfer data one round trip between connections
```

Comment:
```suggestion
    /// Send data from client to server and then from server to client. Used to benchmark throughput of TLS connection.
```

Reply:
I think saying "used to benchmark throughput" isn't that useful, since all of the functions in the harness are used to benchmark something. Also, in the future we might benchmark just `send()` or `recv()`, so `round_trip_transfer()` is just for convenience for now.